### ISSUE TITLE
casia surf test set password

### ISSUE THREAD

**Issue by abaqusstudent** [ZhaoJ9014/face.evoLVe.PyTorch#166]
I have downloaded the casia surf dataset and also managed to unzip the train and val datasets using the password provided in https://github.com/ZhaoJ9014/face.evoLVe.PyTorch/issues/81 but I am unable to unzip the test set. I tried `casia-cefa-test-572121-usd89f` but it says incorrect password.

I would greatly appreciate if someone can provide the password for test.zip

thank you.

rachel
